src/FwdState.cc

```diff
@@ -1349,26 +1349,34 @@ FwdState::reforward()
     return Http::IsReforwardableStatus(s);
 }
 
+// TODO: Refactor to fix multiple mgr:forward accounting/reporting bugs. See
+// https://lists.squid-cache.org/pipermail/squid-users/2024-December/027331.html
 static void
 fwdStats(StoreEntry * s)
 {
     int i;
     int j;
     storeAppendPrintf(s, "Status");
 
-    for (j = 1; j < MAX_FWD_STATS_IDX; ++j) {
+    // XXX: Missing try#0 heading for FwdReplyCodes[0][i]
+    for (j = 1; j <= MAX_FWD_STATS_IDX; ++j) {
         storeAppendPrintf(s, "\ttry#%d", j);
     }
 
     storeAppendPrintf(s, "\n");
 
     for (i = 0; i <= (int) Http::scInvalidHeader; ++i) {
```

Comment:
While this section is being touched, would you mind using WholeEnum for iterating?

Reply:
In this PR, using WholeEnum for this loop is not a good idea, for several reasons. The primary reason is that the underlying data and code do not support at least one value that WholeEnum would include -- scHeaderTooLarge.

Reply:
Another way to reject that WholeEnum idea is to evaluate the first WholeEnum use precondition in EnumIterator.h: "This class requires that ... the underlying enum values be represented by continuous values of an integral type". That precondition is not satisfied by StatusCode enum.